Makefile

```diff
@@ -26,11 +32,16 @@ setup: pipenv
 setup-dev: pipenv
 	$(PIPENV) install --verbose --dev --python $(PYTHON_VERSION)
 	- $(PIPENV_PIP) install flash-attn==2.3.6 --no-build-isolation --upgrade
+	$(PIPENV) run playwright install
 
 .PHONY: setup-no-flash
 setup-no-flash: pipenv
 	$(PIPENV) install --verbose --python $(PYTHON_VERSION)
 
+setup-ui: pipenv
```

Comment:
Why does it need another make command, why is -dev not enough? This should get some coverage in the README.md or any other appropriate docs how the tests are to be used.

Reply:
Probably designed to be run on a remote machine connecting to another instance? Again, should get coverage in docs why it has been designed this way and how it is to be used.

Reply:
precisely, its designed to be able to run from a remote machine. I'll add documentation coverage